src/playground/App.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="main-wrapper">
+    <div class="menu">
+      <router-link to="/">Home</router-link>
+      <router-link to="/marker">Marker</router-link>
+    </div>
+    <div class="map-wrapper">
+      <router-view />
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  components: {},
+};
+</script>
+
+<style scoped>
+.main-wrapper {
+  display: flex;
+}
+.menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.menu a {
+  padding: 0.25rem 0.5rem;
+  text-decoration: none;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.menu .router-link-exact-active {
+  background-color: lightgray;
+}
+.map-wrapper {
+  height: 50vh;
```

Comment:
```suggestion
  height: 75vh;
```
I find a 50vh map becomes _very_ narrow when I open dev tools on the bottom of the browser window. Just personal preference, if narrow is better for others it can stay that way :)